examples/protonect/Protonect.cpp

```diff
@@ -87,6 +91,10 @@ int main(int argc, char *argv[])
     cv::imshow("ir", cv::Mat(ir->height, ir->width, CV_32FC1, ir->data) / 20000.0f);
     cv::imshow("depth", cv::Mat(depth->height, depth->width, CV_32FC1, depth->data) / 4500.0f);
 
+    if (!registered) registered = new unsigned char[depth->height*depth->width*rgb->bytes_per_pixel];
```

Comment:
Given that the depth frame size is already know to be 512x424 if I recall correctly, is there any reason to not to move the allocation out of the loop?